src/utilities.ts

```diff
@@ -9,16 +9,16 @@ const spawn = require("child_process").spawn;
 const hasbin = require("hasbin");
 const fs = require("fs");
 const path = require("path");
+const which = require ("which")
 
 const vPythonName = "vpython";
-const vpythonOnPath = hasbin.sync(vPythonName);
+const vpythonOnPath = which.sync(vPythonName, { nothrow: true })
 export let vPythonCommandToUse: string | undefined = undefined;
 
 const clicastName = "clicast";
```

Comment:
We are not checking if `clicast` is on path anymore, is that by design?

Reply:
yep, I look for the exe after I use vpython to find the install directory